src/components/CellLineInfoCard/CellLineInfoCardBase.tsx

```diff
@@ -1,81 +1,80 @@
-import React, { useState } from "react";
+import React, { ReactNode, useState } from "react";
 import { Card, Flex, Button, Divider, Tooltip } from "antd";
-
-import { PRIMARY_COLOR } from "../style/theme";
-import { formatCellLineId, getCloneSummary } from "../utils";
-import CloneTable from "./CloneTable";
-import { DarkThemeGhostButton, DefaultButton } from "./shared/Buttons";
-import InfoPanel from "./shared/InfoPanel";
 import Icon from "@ant-design/icons";
-import {
-    ParentalLineFrontmatter,
-    GeneFrontMatter,
-    Clone,
-} from "../component-queries/types";
-const Share = require("../img/share-icon.svg");
-const LinkOut = require("../img/external-link.svg");
+
+import { PRIMARY_COLOR } from "../../style/theme";
+import { formatCellLineId } from "../../utils";
+import CloneTable from "../CloneTable";
+import { DarkThemeGhostButton, DefaultButton } from "../shared/Buttons";
+import InfoPanel from "../shared/InfoPanel";
+import { Clone, ParentLine } from "../../component-queries/types";
+
+const Share = require("../../img/share-icon.svg");
+const LinkOut = require("../../img/external-link.svg");
 
 const {
     title,
     container,
     extraLargeButton,
-} = require("../style/cell-line-info-card.module.css");
+} = require("../../style/cell-line-info-card.module.css");
 
-interface CellLineInfoCardProps {
+interface BaseCellLineProps {
     href: string;
+    certificateOfAnalysis: string;
+    healthCertificate: string;
+    orderLink: string;
     cellLineId: number;
     geneName: string;
     geneSymbol: string;
+}
+
+export interface DiseaseProps extends BaseCellLineProps {
+    parentalLine: ParentLine;
     snp: string;
-    orderLink: string;
-    certificateOfAnalysis: string;
-    parentalLine: ParentalLineFrontmatter;
-    parentLineGene: GeneFrontMatter;
     clones: Clone[];
+}
```

Comment:
our convention is to have the props right above and in the same file as the component that is using those props